### ISSUE TITLE
Write accessibility tests for every view in arbeidsflate.

### ISSUE THREAD

**Issue by mbacherycz** [Altinn/dialogporten-frontend#1914]
### Description

We should cover the AF app with accessibility tests to ensure the app has no accessibility violations. 
The initial setup for AF has been configured and the accessibility tests for all components in altinn-components  has been written. 

As of now, we are still updating components from altinn-components library. That makes completing accessibility tests hard as potentially we would have to update it every time we are updating a view with a new component. 
Hence this issue should be resolved AFTER we are done with migrating components from the ac-library. 

Ref: https://github.com/Altinn/dialogporten-frontend/issues/1592
